captura/views.py

```diff
@@ -1,14 +1,23 @@
+from django.contrib.auth.decorators import user_passes_test, login_required
 from django.shortcuts import render
-from django.contrib.auth.decorators import login_required
 
+from perfiles_usuario.utils import is_capturista
+from estudios_socioeconomicos.models import Estudio
 
-@login_required
-def estudios(request):
-    """ DUMMY VIEW.
 
-    This functions is currently just being used to test the redirect
-    from base.
+@login_required
+@user_passes_test(is_capturista)
+def capturista_dashboard(request):
+    """View to render the capturista control dashboard.
 
-    TODO: name properly and implement everything
+       This view shows the list of socio-economic studies that are under review
+       and the button to add a new socio-economic study.
+       Also shows the edit and see feedback buttons to each socio-economic study
+       shown in the list if this exists for the current user (capturist).
     """
-    return render(request, 'administracion/dashboard_users.html')
+    user_id = request.user.id
+    estudios = Estudio.objects.filter(
+            status__in=[Estudio.RECHAZADO, Estudio.REVISION, Estudio.BORRADOR],
+            capturista_id=user_id)
```

Comment:
It is imporant to note that the Capturista is a separate object from the User. If you have just on user in your local database or two it will work, but with multiple users this will become obvious.

You need to query this way:

Estudio.objects.filter(
             status__in=[Estudio.RECHAZADO, Estudio.REVISION, Estudio.BORRADOR],
             capturista=Capturista.objects.get(user=request.user))

In Django you do not need to send ids if you send objects Django knows the way to bind it.